src/memory/AmdFlash.cc

```diff
@@ -555,6 +573,35 @@ bool AmdFlash::checkCommandLongReset()
 	return false;
 }
 
+bool AmdFlash::checkCommandStatusRead()
+{
+	static constexpr std::array<uint8_t, 3> cmdSeq = {0x70};
```

Comment:
I only noticed this now, maybe I overlooked this in previous patches:
The size of the array is specified as 3, but the initializer list only contains 1 element. Is that intentional? I guess not? Or if it is, then it's probably clearer to explicitly write `0x70, 0x00, 0x00`.

Same remark in the next method.

Reply:
Whoops, no definitely not intentional, a copy/pasta.

Now it makes more sense why GCC would [complain about this](https://github.com/openMSX/openMSX/pull/1620#issuecomment-2106403301).

Will fix.